src/Components/UserProfileCard.js

```diff
@@ -1,25 +1,40 @@
 import React from 'react';
-import { Button, Card, Icon, Image } from 'semantic-ui-react';
+import { Grid, Button, Card, Icon, Image } from 'semantic-ui-react';
 
 const UserProfileCard = () => {
   return (
     <Card centered>
-     
       <Card.Content>
-      <Image
-        size="small"
-        floated="left"
-        src="https://react.semantic-ui.com/images/avatar/large/matthew.png"
-      />
-        <Card.Header>Arturo</Card.Header>
-        <Card.Meta>
-          <span className="position">CEO</span>
-        </Card.Meta>
-        <Card.Description>Arturo is a musician living in CDMX.</Card.Description>
-
-        <Button centered color="blue" type="submit">
-          Message
-        </Button>
+        <Grid>
+          <Grid.Column mobile={16} tablet={6} computer={8}>
+            <Grid.Row>
+              <Image
```

Comment:
Try to give the image a circular shape by adding radius or looking into semantic's docs to see if the Image component has a prop for that.